libs/barista-components/filter-field/src/types.ts

```diff
@@ -45,6 +45,7 @@ export interface DtNodeDef<D = unknown> {
 export interface DtAutocompleteDef<OpGr = unknown, Op = unknown> {
   distinct: boolean;
   async: boolean;
+  partial: boolean;
```

Comment:
this needs to be optional as well :) 
```suggestion
  partial?: boolean;
```

Reply:
👍